db/versions/0105.yml

```diff
@@ -0,0 +1,114 @@
+version: 105
+description: insert, fetch and purge audit history for auth, hooks and secrets
+
+migrationScript: 0105-migration.sql
+downgradeScript: 0105-downgrade.sql
+
+methods:
+  insert_auth_audit_history:
+    description: |
+      Insert an audit history entry for a given entity.
+    mode: write
+    serviceName: auth
+    args: entity_id_in text, entity_type_in text, client_id_in text, action_type_in text
+    returns: void
+    body: |-
+      begin
+        INSERT INTO audit_history (
+          entity_id,
+          entity_type,
+          client_id,
+          action_type,
+          created
+        ) VALUES (
+          entity_id_in,
+          entity_type_in,
+          client_id_in,
+          action_type_in,
+          now()
+        );
+      end
+  insert_secrets_audit_history:
+    description: |
+      Insert an audit history entry for a given entity.
+    mode: write
+    serviceName: secrets
+    args: entity_id_in text, entity_type_in text, client_id_in text, action_type_in text
```

Comment:
I think we should drop `entity_type_in` parameter from specialized functions. This way secret service wouldn't be adding values for unrelated entities. 
Same with hooks one. 
It looks a bit strange